homeassistant/components/homematic/__init__.py

```diff
@@ -173,14 +175,17 @@
 DEFAULT_PATH = ''
 DEFAULT_USERNAME = 'Admin'
 DEFAULT_PASSWORD = ''
+DEFAULT_SSL = False
+DEFAULT_VERIFY_SSL = False
```

Comment:
Although I generally agree, for this case (and comparable ones) it wouldn't make sense to verify by default. The reason is, that the CCU-device never is a cloud device, it's always on premise. It can be exposed to the web, but that's a really bad idea. So typically nobody does that. And it's unrealistic, that people will use trusted certificates for a machine that's only local. They might have their custom CA. But last time I checked I couldn't make Home Assistant use a custom CA (virtualenv).
So essentially: having `verify_ssl` with `True` as default will be a broken-by-default configuration.